src/hexo/themes/sonarwhal/layout/partials/metas.hbs

```diff
@@ -10,5 +10,17 @@
 
 <link rel="apple-touch-icon" href="/images/apple-touch-icon.png">
 
-<link rel="icon" type="image/png" href="/images/favicon-32x32.png" sizes="32x32">
-<link rel="icon" type="image/png" href="/images/favicon-16x16.png" sizes="16x16">
+{{#if status}}
+    {{#if (isFinish status)}}
+        {{#if (isPass status overallStatistics)}}
+            <link rel="icon" type="image/png" class="favicon" href="/images/favicon_passed.ico" sizes="32x32">
```

Comment:
Since we are using just one favicon, we can drop `sizes`, also `type` can be removed (it should be `image/x-icon` anyway :P).